netmanager/src/views/pages/UserList/components/UsersToolbar/UsersToolbar.js

```diff
@@ -96,16 +96,15 @@ const UsersToolbar = (props) => {
  const orgData = useOrgData();
 
   const initialState = {
-    userName: "",
     firstName: "",
     lastName: "",
     organization: orgData.name,
+    long_organization: orgData.name,
```

Comment:
Currently, we dont have the `long_org` data (it's still work in process). The tenant key might change from `name` to `tenant` too. For now since the `long_org` is a required field when creating a user, it defaults to the name/tenant for now. Changes will be made accordingly when all the PRs/ DB changes are ready.